bin/trendAnalyse.sh

```diff
@@ -244,6 +244,84 @@ function processProjectToDB() {
 
 }
 
+function processRNAProjectToDB {
+	local _rnaproject="${1}"
+	local _processrnaprojecttodb_controle_line_base="${2}"
+	
+	CHRONQC_TMP="${TMP_TRENDANALYSE_DIR}/tmp/"
+	CHRONQC_RNAPROJECTS_DIR="${TMP_TRENDANALYSE_DIR}/RNAprojects/${_rnaproject}/"
+	CHRONQC_DATABASE_NAME="${TMP_TRENDANALYSE_DIR}/database/"
+
+	log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Removing files from ${CHRONQC_TMP} ..."
+	#rm -rf "${CHRONQC_TMP:-missing}"/*
+
+	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "__________processing ${_rnaproject}.run_date_info.csv_____________"
+	if [[ -e "${CHRONQC_RNAPROJECTS_DIR}/${_rnaproject}.run_date_info.csv" ]]
+	then
+		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "found ${CHRONQC_RNAPROJECTS_DIR}/${_rnaproject}.run_date_info.csv. Updating ChronQC database with ${_rnaproject}."
+		cp "${CHRONQC_RNAPROJECTS_DIR}/${_rnaproject}.run_date_info.csv" "${CHRONQC_TMP}/${_rnaproject}.run_date_info.csv"
+		for RNAmultiQC in "${MULTIQC_RNA_METRICS_TO_PLOT[@]}"
+		do
+	#'multiqc_general_stats.txt:general_stats'
+	#'multiqc_star.txt:star'
+	#'multiqc_picard_RnaSeqMetrics.txt:RnaSeqMetrics'
+
+			local _rnametrics="${RNAmultiQC%:*}"
+			log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "using _rnametrics: ${_rnametrics}"
+			if [[ "${_rnametrics}" == multiqc_picard_RnaSeqMetrics.txt ]]
+			then
+				cp "${CHRONQC_RNAPROJECTS_DIR}/${_rnametrics}" "${CHRONQC_TMP}/${_rnaproject}.${_rnametrics}"
+				perl -pe 's|SAMPLE\t|SAMPLE_NAME2\t|' "${CHRONQC_TMP}/${_rnaproject}.${_rnametrics}" > "${CHRONQC_TMP}/${_rnaproject}.1.${_rnametrics}"
+			else
+				cp "${CHRONQC_RNAPROJECTS_DIR}/${_rnametrics}" "${CHRONQC_TMP}/${_rnaproject}.${_rnametrics}"
+			fi
```

Comment:
klopt het dat je in deze else, twee soorten metrix files kopieerd naar dezelfde folder? de star metrix files en de general_Stats?

Reply:
klopt